src/collection.js

```diff
@@ -79,13 +79,18 @@ export default class Collection {
    *
    * @param  {Object} [options={}]      The options object.
    * @param  {Object} [options.headers] The headers object option.
+   * @param  {String} [options.since]   The since ETag option.
    * @return {Promise<Number, Error>}
    */
   getTotalRecords(options={}) {
     const { headers } = this._collOptions(options);
+    let path = endpoint("record", this.bucket.name, this.name);
+    if(options.since){
+      path += "?since="+options.since;
```

Comment:
It may be nice to reuse the `utils.qsify` function here.

Also nit: space around operand